tests/blockchain/blockchain_test_utils.py

```diff
@@ -59,7 +59,10 @@ async def _validate_and_add_block(
     else:
         # Do not change this, validate_signatures must be False
         pre_validation_results: List[PreValidationResult] = await blockchain.pre_validate_blocks_multiprocessing(
-            [block], {}, validate_signatures=False
+            [block],
+            {},
+            validate_signatures=False,
+            fork_height=uint32(0) if fork_point_with_peak is None else fork_point_with_peak,
```

Comment:
`fork_height` is `Optional`. it looks suspicious to convert a `None` into `uint32(0)` here. Why not just forward `fork_point_with_peak` as is?